docs/guides/syslog-service-integration.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+parent: Guides
+title: Creating a Syslog Service Integration
+nav_order: 5
+---
+
+# Creating a Syslog Service Integration
+
+Let's say you have a project in your account and a Kafka service that is throwing all sorts of errors. The Aiven web console has logs built in, you cry! Sure, but you likely need the search power of Elasticsearch or the power of logging services, such as Datadog or Rsyslog servers.
+
+In Terraform, we support adding these service integrations and it can be broken down into 2 steps:
+
+1. Creating an `aiven_service_integration_endpoint` - This will actually create the Service Integration that can be found in the `Service Integration` panel in the Web Console.
+2. Creating an `aiven_service_integration` - This will link the endpoint to a running Aiven service.
+
+## Example
+
+In this example, we already have a project (`my-proj`) and a service (`kafka-service1`) running. This means we can use the `datasource` to pull in those objects from Aiven.
+
+Then, we define the `aiven_service_integration_endpoint`.
+
+The important things here are:
+* endpoint_type - The type of integration (e.g. `rsyslog`, `prometheus`, `rsyslog`)
```

Comment:
`rsyslog` is mentioned two times here, maybe we can replace one with `datadog` for example.

Reply:
Thanks, done